requirements.txt

```diff
@@ -48,7 +48,6 @@ gevent>=1.3.6
 
 # For text normalization
 ftfy
-word2number>=1.1
```

Comment:
The RC code hasn't been removed yet, has it?

Reply:
These are all done in #3348.

Reply:
Oops, sorry.

Reply:
Fair play, I noticed because the SQL ones got missed from the semparse stuff. i'll close this